github/template.go

```diff
@@ -0,0 +1,77 @@
+package github
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/github/hub/utils"
+)
+
+const (
+	pullRequestTempalte = "PULL_REQUEST_TEMPLATE"
+	issueTempalte       = "ISSUE_TEMPLATE"
+	githubTemplateDir   = ".github"
+)
+
+func GetPullRequestTempalte() string {
+	return getGithubTemplate(pullRequestTempalte)
+}
+
+func GetIssueTempalte() string {
```

Comment:
There's a typo here. Should be `GetIssueTemplate `

Reply:
![Feature](https://user-images.githubusercontent.com/51739397/129476750-d71aa97e-7c76-4def-8d00-f229362f4bc5.png)